test/resources/case20_delete_objects/case20_edit_pod.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: policy.open-cluster-management.io/v1
+kind: ConfigurationPolicy
+metadata:
+  name: policy-pod-edit
```

Comment:
Optional nit: I wonder if a more accurate name would be `policy-pod-already-created`.